pylibui/controls/box.py

```diff
@@ -34,6 +26,35 @@ def append(self, child, stretchy=0):
         :return: None
         """
         libui.uiBoxAppend(self.control, child.pointer(), stretchy)
+        self.children.append(child)
+
+    def delete(self, index):
+        """
+        Deletes a child from a box.
+
+        :param index: int
+        :return: None
+        """
+        libui.uiBoxDelete(self.control, index)
+        self.children[index].destroy()
+        del self.children[index]
+
+    def getPadded(self):
+        """
+        Returns whether the box is padded.
```

Comment:
Hum, ok, this is in pylibui/controls, so we can design our own API if we wanted. For instance we could change the function name to be better, such as "isPadded", for instance.

But let's leave it for now as it is, just change the return docstring to "boolean" instead of "int" and it will be all right..